### ISSUE TITLE
Alluxio master metrics web page hung for several minutes

### ISSUE THREAD

**Issue by maobaolong** [Alluxio/alluxio#13657]
**Alluxio Version:**
master branch 

**Describe the bug**
Open metrics page of master `http://localhost:19999/metrics` cost several minutes while the master server running over 100 days. We should record only enough point for dataPoints of TimeSeries

**To Reproduce**

Running a master over 100 days.
Open  `http://localhost:19999/metrics` or open `http://localhost:8081/api/v1/master/webui_metrics`

![image](https://user-images.githubusercontent.com/17329931/122342102-c994e100-cf76-11eb-9e24-a59ab95a19a6.png)

We only use the latest 20 points of data.

![image](https://user-images.githubusercontent.com/17329931/122342335-18db1180-cf77-11eb-91b5-200dfe5b20e3.png)

**Expected behavior**

Open page in a second, but wait for minutes.

**Urgency**

High

**Additional context**

We should limit the size of `TimeSeries#mDataPoints` 

```java
public class TimeSeries {
  private final List<DataPoint> mDataPoints;
...
}

```

**Comment by LuQQiu** [comment]
@tieujason330 can you help take a look this issue, thanks

**Comment by apc999** [comment]
Fixed by #13660 (thanks for @tieujason330 )
Closing this issue and feel free to re-open it if you still it happening.
